docs/radio.rst

```diff
@@ -0,0 +1,67 @@
+Radio
+*****
+
+.. py:module:: radio
+
+The ``radio`` module allows devices to work together via simple wireless
+networks.
+
+Functions
+=========
+
+.. py:function:: send(message)
+
+    Sends a message, expressed as a string.
+
+.. py:function:: receive()
+
+    Receive the next incoming message on the message queue. Returns ``None`` if
+    there are no pending messages.
```

Comment:
We should specify the type of the returned value, ie bytes.  It should be bytes because that's what goes through the air.